### ISSUE TITLE
[BUG] Documentation is misleading.

### ISSUE THREAD

**Issue by mahyarmirrashed** [lidatong/dataclasses-json#530]
### Description

Documentation says that the latest release is available for Python3.6 but, it is not. Python3.6 support was removed here, on March 18, 2023: https://github.com/lidatong/dataclasses-json/commit/77526873ade0424561c60416a3bcb29dde92b302.
I am building on a frozen system that is unable to upgrade its Python version. Otherwise, this wouldn't be much of an issue since Python3.6 is EOL anyway.
I'd just like to see the text in the README.md where it says that 3.6 is supported to be removed. Thanks.

### Code snippet that reproduces the issue

N/A

### Describe the results you expected

To 

### Python version you are using

3.6.9

### Environment description

pip==9.0.1

**Comment by george-zubrienko** [comment]
@mahyarmirrashed you are welcome to submit a PR to correct the docs :)

**Comment by mahyarmirrashed** [comment]
@george-zubrienko you're correct! I just wanted to raise it with you before I did something like that. Thanks for confirming, I'll create a PR soon.

**Comment by mahyarmirrashed** [comment]
@george-zubrienko I've created a PR, please review at your earliest convenience.
